docs/pages/tutorial/eas/ios-development-build-for-devices.mdx

```diff
@@ -0,0 +1,229 @@
+---
+title: Create and run a cloud build for iOS device
+sidebar_title: iOS development build for devices
+description: Learn how to configure a development build for iOS devices using EAS Build.
+---
+
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, let's create our development build for iOS devices with EAS Build.
+
+## Trigger a build
+
+Development builds for iOS devices are generated in the **.ipa** format, which is standard for iOS app installations.
+
+> **Note**: Apple Developer membership is required to create and install a development build on an iOS device.
+
+<Step label="1">
+
+## Register an iOS device with a provisioning profile
+
+To initiate development on an iOS device, we have to:
+
+- Register the device by creating a new [provisioning profile](/app-signing/app-credentials/#provisioning-profiles).
+- Download and install this profile onto the device.
+
+With EAS CLI, run the command to register a new Apple device:
+
+<Terminal cmd={['$ eas device:create']} />
+
+This command prompts us with the following questions:
+
+- **You're inside the project directory. Would you like to use the** **your-account-name** **account?** Press <kbd>Y</kbd>.
+- **Apple ID.** For this step, enter your Apple ID. It will then log in to our Apple Developer account. Follow the steps in the terminal window.
+- **How would you like to register your devices?** Select **Website** that generates a registration URL that can be opened on the iOS device.
+
+> **info** **Tip**: If you or your team have multiple devices, you can share the provisioning profile link with those devices for downloading and installing the profile.
+
+</Step>
+
+<Step label="2">
+
+## Download and install the provisioning profile
```

Comment:
Hm, you know I've never used that flow before so I can't actually comment on it. I keep it simple by registering the device with the UDID and adding it to the profile. Maybe we can provide that as an alternative?

Reply:
"I keep it simple by registering the device with the UDID and adding it to the profile"

- Can you elaborate on this more or give me tips so that I can try it myself? I don't think I've used this method.

Reply:
Haha, maybe it's best to keep yours then!

But to explain, UDID is basically this (you can see it in Finder when you plug your phone in). It's a unique ID for your device. You copy it, and if you choose the UDID option it asks you to name the device and add that ID. Then you add that device to the Ad Hoc provisioning profile and that's it.


<img width="1032" alt="Screenshot 2024-02-21 at 20 15 35" src="https://github.com/expo/expo/assets/6534400/f778b440-e704-4b7b-b43b-83e371c9e37b">

Reply:
Thanks for sharing these details! I think you are right about adding this as an alternative. I'll try this on my end and just to be able to document the steps in this process. Will follow up on this separately (maybe as a separate task a bit later).